pulp_container/tests/functional/api/test_rbac_repo_versions.py

```diff
@@ -271,17 +271,18 @@ def setUpClass(cls):
         cls.distribution = cls.distributions_api.list(name="test-1").results[0]
 
         cls.user_pull = gen_user(
-            object_permissions=[
-                ("container.pull_containerdistribution", cls.distribution.pulp_href)
+            object_roles=[
+                ("container.containerdistribution_consumer", cls.distribution.pulp_href),
+                ("container.containernamespace_consumer", cls.distribution.namespace),
```

Comment:
Would not giving them the namespace role be sufficient? (I think it i should be.)